Aztec/Classes/TextKit/TextView.swift

```diff
@@ -1000,7 +1000,29 @@ open class TextView: UITextView {
     ///
     open func refreshLayoutFor(attachment: TextAttachment) {
         layoutManager.invalidateLayoutForAttachment(attachment)
-    }    
+    }
+
+
+    // MARK: - More
+
+    /// Inserts the More Comment at the specified position.
+    ///
+    /// - Parameter range: The character range that must be replaced by a More Attachment.
+    ///
+    /// - Returns: the attachment object that can be used for further calls
+    ///
+    @discardableResult
+    open func replaceRangeWithMoreAttachment(_ range: NSRange) -> MoreAttachment {
+        let attachment = storage.replaceRangeWithMoreAttachment(range)
+        let attachmentRange = NSRange(location: range.location, length: NSAttributedString.lengthOfTextAttachment)
```

Comment:
Instead of inserting the "more attachment" and then calculating the range to set the styles, why don't you change it so that you just call: `storage.replaceRangeWithMoreAttachment(range, attributes: attributes)`?

The logic to set the style for a newly added "more attachment" would be useful to anyone calling `replaceRangeWithMoreAttachment()`.